### DISCUSSION TITLE
Identity micro service

### DISCUSSION THREAD

**Discussion by samidarko** [fastapi-users/fastapi-users#594] [Show and tell]
Hi,

I made a docker image with `fastapi-users`. It makes a good base for an authentication micro-service . I don't know if it can be useful to anyone except me but I thought of sharing what I did here. Thanks for the awesome library!

https://hub.docker.com/r/samidarko/identity

https://github.com/samidarko/identity

Feedback are welcome :)

**Comment by frankie567** [comment]
Hi @samidarko!

Thank you for sharing this with us, that's great! I think it'll also help people to configure their project correctly 😄

I see that you are planning to add a database migration system, I would love to see that!

**Reply by samidarko** [reply]
I integrated [aerich](https://github.com/tortoise/aerich) to the project :)
